snmp/datadog_checks/snmp/data/profiles/_base_hp.yaml

```diff
@@ -0,0 +1,190 @@
+metrics:
+  # System utilization.
+
+  - # Total time the system has been in full operation.
+    MIB: CPQHLTH-MIB
+    symbol:
+      OID: 1.3.6.1.4.1.232.6.2.8.1
+      name: cpqHeSysUtilLifeTime
+
+  - # EISA bus utilization as percentage of theoretical maximum during last minute.
+    MIB: CPQHTLH-MIB
+    symbol:
+      OID: 1.3.6.1.4.1.232.6.2.8.2
+      name: cpqHeSysUtilEisaBusMin
+
+  # Critical errors.
+
+  - # Overall condition of the critical error log feature.
+    # NOTE: other(1), ok(2), degraded(3), failed(4)
+    MIB: CPQHTLH-MIB
+    symbol:
+      OID: 1.3.6.1.4.1.232.6.2.2.2
+      name: cpqHeCritLogCondition
+
+  # Correctable memory errors tracking.
+
+  - # Overall status of the correctable memory error log feature.
+    # NOTE: other(1), notSupported(2), disabled(3), enabled(4)
+    MIB: CPQHLTH-MIB
+    symbol:
+      OID: 1.3.6.1.4.1.232.6.2.3.1
+      name: cpqHeCorrMemLogStatus
+
+  - # Overall condition of the correctable memory error log feature.
+    # NOTE: other(1), ok(2), degraded(3), failed(4)
+    MIB: CPQHLTH-MIB
+    symbol:
+      OID: 1.3.6.1.4.1.232.6.2.3.2
+      name: cpqHeCorrMemLogCondition
+
+  - # Number of correctable memory errors that have occurred.
+    MIB: CPQHLTH-MIB
+    symbol:
+      OID: 1.3.6.1.4.1.232.6.2.3.3
+      name: cpqHeCorrMemTotalErrs
+    forced_type: monotonic_count
+
+  # ASR (Automatic Server Recovery)
+
+  - # Overall status of the ASR feature.
+    # NOTE: other(1), notAvailable(2), disabled(3), enabled(4)
+    MIB: CPQHLTH-MIB
+    symbol:
+      OID: 1.3.6.1.4.1.232.6.2.5.1
+      name: cpqHeAsrStatus
+
+  - # Whether the ASR timer passed the server Power-On Self-Test (POST).
+    # NOTE: other(1), failed(2), ok(3)
+    MIB: CPQHLTH-MIB
+    symbol:
+      OID: 1.3.6.1.4.1.232.6.2.5.6
+      name: cpqHeAsrPost
+
+  - # Number of ASR reboots that occurred on the server since last manual reboot.
+    # NOTE: -1 if not supported.
```

Comment:
with monotonic_count, there's no negative values, should we leave this type as gauge? https://docs.datadoghq.com/developers/metrics/agent_metrics_submission/?tab=count#monotonic-count

Reply:
I agree it's a weird case. Since this is an edge case where it is either supported or not (ie a server that supports it probably won't stop supporting it, and if it does then points submitted afterwards will be ignored), do you think it's safer to not change the type?